lxd/storage_volumes_backup.go

```diff
@@ -424,9 +423,9 @@ func storagePoolVolumeTypeCustomBackupsPost(d *Daemon, r *http.Request) response
 		return nil
 	}
 
-	resources := map[string][]string{}
-	resources["storage_volumes"] = []string{volumeName}
-	resources["backups"] = []string{req.Name}
+	resources := map[string][]api.URL{}
+	resources["storage_volumes"] = []api.URL{*api.NewURL().Path(version.APIVersion, "storage-pools", poolName, "volumes", volumeTypeName, volumeName, "backups")}
```

Comment:
This isnt the correct URL. Lose the "backups" from the end, as pointing to a list of backups is neither a resource nor the correct resource type for storage_volumes.